kibble/models/film.go

```diff
@@ -105,41 +102,41 @@ func (film Film) GetSubtitles() StringCollection {
 }
 
 // MakeTitleSlugsUnique scans the films looking for duplicates
-func (films *FilmCollection) MakeTitleSlugsUnique() {
-
-	groups := make(map[string][]int, 0)
-
-	// create a grouping of slugs to films first
-	for fi, film := range *films {
-		if groups[film.TitleSlug] == nil {
-			groups[film.TitleSlug] = []int{fi}
-		} else {
-			groups[film.TitleSlug] = append(groups[film.TitleSlug], fi)
-		}
-	}
-
-	// if any groups are larger than 1 then make them unique
-	for _, group := range groups {
-		if len(group) == 1 {
-			continue
-		}
-
-		// sort them by id, so the first film is not changed
-		sort.Slice(group, func(i int, j int) bool {
-			return (*films)[group[i]].ID < (*films)[group[j]].ID
-		})
-
-		// append i + 1 to end of slug
-		for j := 0; j < len(group); j++ {
-			if j == 0 {
-				continue
-			}
-
-			// find the correct value to update, without making a copy
-			(*films)[group[j]].TitleSlug = fmt.Sprintf("%s-%d", (*films)[group[j]].TitleSlug, j+1)
-		}
-	}
-}
+// func (films *FilmCollection) MakeTitleSlugsUnique() {
```

Comment:
Will removing this make films with identical titles inaccessible?

Reply:
re-added